src/test/resources/pacts/publicapi-connector-search-payment-by-first-digits-card-number.json

```diff
@@ -48,6 +48,18 @@
               "description": "Test description",
               "reference": "aReference",
               "language": "en",
+              "links": [
```

Comment:
We will also need similar change in
`publicapi-connector-search-payment-by-cardholder-name`
`publicapi-connector-search-payment-by-last-digits-card-number`

Reply:
Apologies I did a rebase and it lost those changes. I've now updated it now.